pkg/policy/execute.go

```diff
@@ -238,24 +237,14 @@ func (e *Executor) executeQuery(ctx context.Context, q *Query) (*QueryResult, er
 // createViews creates temporary views for given config.Policy, and any views defined by sub-policies
 func (e *Executor) createViews(ctx context.Context, policy *Policy) error {
 	for _, v := range policy.Views {
-		e.log.Debug("creating policy view", "view", v.Name)
-		if err := e.createView(ctx, v); err != nil {
-			return fmt.Errorf("%s/%s/%w", policy.Name, v.Name, err)
+		e.log.Info("creating policy view", "view", v.Name)
+		if _, err := e.conn.Exec(ctx, fmt.Sprintf("CREATE OR REPLACE TEMPORARY VIEW %s AS %s", v.Name, v.Query)); err != nil {
```

Comment:
> Temporary views are automatically dropped at the end of the current session

If we're using a connection pool and the the CREATE and the querying runs on different conns on the pool, we'd encounter 404 errors from the DB.

Reply:
I don't think ill solve this in this PR